package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "avif-webpack-loader",
+  "version": "0.1.0",
+  "description": "Webpack loader for AVIF-encoded images",
+  "author": "Maxim Shmakov <mvshmakov@mail.ru>",
+  "license": "MIT",
+  "keywords": [
+    "avif",
+    "av1",
+    "webpack",
+    "loader",
+    "images"
+  ],
+  "homepage": "https://github.com/mvshmakov/avif-webpack-loader",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mvshmakov/avif-webpack-loader.git"
+  },
+  "bugs": {
+    "url": "https://github.com/mvshmakov/avif-webpack-loader/issues"
+  },
+  "main": "src/loader.js",
+  "type": "module",
+  "engines": {
+    "node": ">=10.13.0"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "lint:check": "eslint src/*.{js,json} test/*.js",
+    "lint:fix": "eslint --fix src/*.{js,json} test/*.js",
+    "prettier:check": "prettier --check src/*.{js,json} test/*.js",
+    "prettier:fix": "prettier --write src/*.{js,json} test/*.js",
+    "test": "RUN_IN_MEMORY=true jest --runInBand"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "npm run lint:fix",
+      "npm run prettier:fix",
+      "npm run test",
+      "git add ."
+    ]
+  },
+  "dependencies": {
+    "@saschazar/wasm-avif": "^1.0.0",
+    "@saschazar/wasm-image-loader": "^3.0.0",
+    "loader-utils": "^2.0.0",
```

Comment:
We don't need to have these packages in the dependencies. They should be moved to `devDependencies`.